cedar-policy/src/ffi/mod.rs

```diff
@@ -14,9 +14,11 @@
  * limitations under the License.
  */
 
+#![cfg_attr(feature = "wasm", allow(non_snake_case))]
```

Comment:
Out of curiosity: what is this for? I thought wasm preferred camelCase to snake_case?

Reply:
Wasm does prefer camelCase.  Including, some of tsify's macros expand to functions with camelCase names, which clippy doesn't like.  This `allow` avoids warnings for these autogenerated camelCased functions.